src/main/kotlin/no/nav/helsearbeidsgiver/domene/inntektsmelding/Inntektsmelding.kt

```diff
@@ -1,42 +1,23 @@
-@file:UseSerializers(LocalDateSerializer::class, OffsetDateTimeSerializer::class)
+@file:UseSerializers(OffsetDateTimeSerializer::class, UuidSerializer::class)
 
 package no.nav.helsearbeidsgiver.domene.inntektsmelding
 
-import kotlinx.serialization.EncodeDefault
-import kotlinx.serialization.ExperimentalSerializationApi
 import kotlinx.serialization.Serializable
 import kotlinx.serialization.UseSerializers
-import no.nav.helsearbeidsgiver.utils.json.serializer.LocalDateSerializer
 import no.nav.helsearbeidsgiver.utils.json.serializer.OffsetDateTimeSerializer
-import java.time.LocalDate
+import no.nav.helsearbeidsgiver.utils.json.serializer.UuidSerializer
 import java.time.OffsetDateTime
+import java.util.UUID
 
 @Serializable
-@OptIn(ExperimentalSerializationApi::class)
 data class Inntektsmelding(
-    val orgnrUnderenhet: String,
-    val identitetsnummer: String,
-    val fulltNavn: String,
-    val virksomhetNavn: String,
-    val behandlingsdager: List<LocalDate>,
-    val egenmeldingsperioder: List<Periode>,
-    val fraværsperioder: List<Periode>,
-    val arbeidsgiverperioder: List<Periode>,
-    val beregnetInntekt: Double,
-    val inntektsdato: LocalDate? = null,
-    val inntekt: Inntekt? = null,
-    val fullLønnIArbeidsgiverPerioden: FullLoennIArbeidsgiverPerioden? = null,
+    val id: UUID,
+    val sykmeldt: Sykmeldt,
```

Comment:
Kanskje bruke "arbeidstaker", heller enn sykmeldt?

Reply:
Andre steder hvor det er brukt "arbeidstaker" og "arbeidsgiver" så synes jeg de for like og lette å ta feil av.

Reply:
Man må kunne forskjellen på gi og ta  :P 
Nå heter vel arbeidsgiver "innsender" tilogmed, så synes ikke det er så likt.
Og det finnes jo inntektsmeldinger for folk som ikke er sykmeldt også..